webui/react/src/pages/Clusters/ClustersQueuedChart.tsx

```diff
@@ -32,19 +32,29 @@ const ClustersQueuedChart: React.FC<Props> = ({ poolStats }: Props) => {
     return [now - viewDays * 86400, now];
   }, [viewDays]);
 
+  const options = [
```

Comment:
Do we want this border? We dont have that in the current radio buttons

<img width="177" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/0cf8e5ce-23b6-49c2-b155-d0a8522e6142">